### DISCUSSION TITLE
Does FORCE support Copernicus Atmosphere Monitoring Service (CAMS) data?

### DISCUSSION THREAD

**Discussion by ffoss** [davidfrantz/force#332] [Q&A]
Hello David,

It is unclear from the documentation, and the parameter file if FORCE supports CAMS data. Can you help me out here?

/K

**Comment by Florian-Katerndahl** [comment]
Leaving a comment to stay in the loop on this one 🙂 

A while back, I implemented a cli tool to query and download CAMS data. The endgoal was to integrate this into FORCE. However, I never came about to actually do so due to my bachelor thesis, studies and work. Curious, if there was any further work done on this.

Might be interesting to pick this up again, if wanted and time permits.

Cheers
Florian

**Reply by davidfrantz** [reply]
Hi Florian,

It would still be a worthwile and welcome addition! Though colleagues from our meteorological department suggested to rather use ERA-5 (would have the advantage of going back all the way to the early Landsats). The general strategy would be the same though.

Let me know if you want to look into this again!

Cheers,
David

**Reply by Florian-Katerndahl** [reply]
Hey David,
I would be very interested to do so! However, I won't make any promises as to when (or whether at all) I have the time to take a detailed look into ERA-5 and an integration into FORCE. Sadly, time remains scarce as I recently picked up other engagements that will likely become time consuming in the near future.

Cheers
Florian

**Reply by davidfrantz** [reply]
Great! And no worries: any contribution is welcome. Be it feature contributions, bugfixes, or just answering a question here and there and being a part of this community!

**Comment by davidfrantz** [comment]
@ffoss can you please elaborate on what you have planned?

As an alternative to the MODIS water vapor data, or are you interested in processing CAMS data?

Cheers,
David

**Reply by ffoss** [reply]
Hi again. 

Sorry that I missed the notification from the thread. We primarily had the curiosity to use CAMS data as an alternative and wanted to know if FORCE already has something implemented. We have put this question on ice for now, and plan to revisit at the end of this year.

Regards,
Kabir

**Comment by davidfrantz** [comment]
@ffoss. alright, I am closing this now. Feel free to open a new discussion later this year.

Best,
David
